file.txt

```diff
@@ -0,0 +1 @@
+hi this is veer
```

Comment:
Purpose of this file?

Reply:
Forgot to remove that,  I was testing Ajax load to get data from another files(server).